hieradata/nodes/lowgpa.yaml

```diff
@@ -0,0 +1,5 @@
+classes:
+    - ocf_decal
+
+ocf::auth::glogin: [[ocfdecal, ALL]]
```

Comment:
this LDAP group remains to be made

Reply:
Are we still going to make this if we use DigitalOcean VMs for the advanced track?